CHANGELOG.rst

```diff
@@ -2,14 +2,141 @@
 Changelog
 =========
 
+2.7.0-alpha.0 (2024-01-05)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+**New features**
+
+Improved backend validation robustness, mainly by validating new components:
+
+* [#4035] Added an E2E test for the file component.
+* [#72] Added validation for the following components: time, selectboxes, textarea, postcode, bsn, select, checkbox,
+  currency, signature, map, cosign, password, iban and licenseplate.
+
+Other features:
+
+* [#4031] Added a confirm popup when switching to v2 options.
+* [#4041] Improved API documents registration robustness, taking into account possible crashes when sending the documents.
+* [#3969] Removed the ability to override LoA for eHekrenning/eIDAS.
+* [#4009, #979] Removed read-only "Merged data" submission field in the admin (related to some internal code cleanup).
+* [#4005] Added the ability to search submission reports by public registration reference and submission in the admin.
+* [#4005] Updated title of the PDF submission report to include the public registration reference.
+* [#3725] Improved email digest by detecting more problems. This includes:
+
+   - Submissions with failed registration status.
+   - Log failures.
+   - Working configuration of the BRK client if the ``addressNL`` component is used.
+   - Working configuration of the BAG API if deriving street/city from postal code is used.
```

Comment:
"Address autofill (based on postal code and house numer) misconfiguration" maybe?